gradle-plugin/doc/src/main/asciidoc/inc/scenarios/_quarkus.adoc

```diff
@@ -0,0 +1,116 @@
+[[quarkus-scenario]]
+== Get started with Quarkus and Minikube
+You can easily get started with using {plugin} on a https://quarkus.io/[Quarkus] without providing any explicit configuration. {plugin} would generate an opinionated container image and manifests by inspecting your project configuration.
```

Comment:
`You can easily get started with using {plugin} [...]`

I think there might be some word missing in between with and using.

Or maybe just remove using.